inference/core/env.py

```diff
@@ -254,3 +254,7 @@
 
 ACTIVE_LEARNING_ENABLED = str2bool(os.getenv("ACTIVE_LEARNING_ENABLED", False))
 ACTIVE_LEARNING_TAGS = safe_split_value(os.getenv("REQUIRED_ONNX_PROVIDERS", None))
+
+NUM_PARALLEL_TASKS = os.getenv("NUM_PARALLEL_TASKS", 1000)
```

Comment:
Is this the maximum number of requests that can be "in the system" at any one time? In other words, the maximum number of web workers that are waiting for an inference result?

Reply:
Number of inflight tasks. There's no maximum to the number of web workers, but those workers will just be waiting around sleeping until theres less than this number of task keys available